norfair/drawing.py

```diff
@@ -195,13 +197,14 @@ class Color:
     blue = (255, 0, 0)
     teal = (128, 128, 0)
     silver = (192, 192, 192)
+    rand = (-1, -1, -1) #random color for each detection
```

Comment:
Piggy backing on my last comment, this is where the random logic should be defined, as a `@property`.